tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,95 @@
+from robottelo.constants import (
+    ANY_CONTEXT,
+    OSCAP_CONTENT_FILE,
+    OSCAP_DEFAULT_CONTENT,
+)
+from robottelo.datafactory import valid_data_list
+from robottelo.helpers import get_data_file
+from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context, make_oscapcontent
+from robottelo.ui.session import Session
+
+
+class OpenScapContent(UITestCase):
+    """Implements Oscap Content tests in UI."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(OpenScapContent, cls).setUpClass()
+        cls.content_path = get_data_file(OSCAP_CONTENT_FILE)
+
+    def test_positive_create_oscapcontent(self):
+        """@Test: Create OpenScap content.
+
+        @Feature: OpenScap - Positive Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is successful.
+
+        """
+        with Session(self.browser) as session:
+            for content_name in valid_data_list():
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(
+                        self.oscapcontent.search(content_name))
+
+    def test_positive_default_oscapcontent(self):
+        """@Test: Check whether OpenScap content exists by default.
+
+        @Feature: OpenScap - Test Default Content.
+
+        @Steps:
+
+        1. Navigate to oscap Content page.
```

Comment:
Can you add this new module to documentation and run `make docs-clean docs` to check if it will raise any warnings related to this module?